contents/tutorials/vue-analytics.md

```diff
@@ -0,0 +1,279 @@
+tags: ['experimentation']
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+import EventsLight from '../images/tutorials/vue-surveys/events-light.png'
+import EventsDark from '../images/tutorials/vue-surveys/events-dark.png'
+
+[Product analytics](/product-analytics) enable you to gather and analyze data about how users interact with your Vue.js app. To show you how to set up analytics, in this tutorial we create a basic Vue app, add PostHog, and show you have to capture pageviews and custom events.
+
+## Creating a Vue app
+
+To demonstrate the basics of PostHog analytics, we'll create a simple Vue 3 app with two pages and a link to navigate between them.
+
+First, ensure [Node.js is installed](https://nodejs.dev/en/learn/how-to-install-nodejs/) (version 14.6.0 or newer). Then install `@vue/cli` and create a new Vue app:
+
+```bash
+npm install -g @vue/cli
+vue create vue-analytics
+```
+
+Make sure to select `[Vue 3] babel, eslint` as the Vue version.
+
+### Adding pages
+
+Next, create two new files `HomePage.vue` and `AboutPage.vue` in `src/components`. 
+
+In `HomePage.vue`, add the following the code:
+
+```vue file=components/HomePage.vue
+<template>
+  <div>
+    <h1>Home</h1>
+    <router-link to="/about">Go to About</router-link>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'HomePage'
+}
+</script>
+```
+
+In `AboutPage.vue`, add the following the code:
+
+```vue file=components/AboutPage.vue
+<template>
+  <div>
+    <h1>About</h1>
+    <router-link to="/">Back Home</router-link>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'AboutPage'
+}
+</script>
+```
+
+Next we set up the routing. First install Vue Router in your project:
+
+```bash
+npm install vue-router@4
+```
+
+In your `src` directory, create a new folder `router` and then a new file `index.js`:
+
+```bash
+cd ./src
+mkdir router
+cd router 
+touch index.js
+```
+
+In `router/index.js`, import the components and define the routes:
+
+```js file=router/index.js
+import { createRouter, createWebHistory } from 'vue-router'
+import HomePage from '../components/HomePage.vue'
+import AboutPage from '../components/AboutPage.vue'
+
+const routes = [
+  {
+    path: '/',
+    name: 'HomePage',
+    component: HomePage
+  },
+  {
+    path: '/about',
+    name: 'AboutPage',
+    component: AboutPage
+  }
+]
+
+const router = createRouter({
+  history: createWebHistory(),
+  routes
+})
+
+export default router
+```
+
+Then, update your `App.vue` to use the router-view. Replace the contents of this file with:
+
+```vue file=App.vue
+<template>
+  <router-view></router-view>
+</template>
+
+<script>
+export default {
+  name: 'App'
+}
+</script>
+```
+
+Finally, set up `main.js` to use the router:
+
+```js file=main.js
+import { createApp } from 'vue'
+import App from './App.vue'
+import router from './router'
+
+createApp(App).use(router).mount('#app')
+```
+
+The basic setup is now complete. Run `npm run serve` to see your app.
```

Comment:
If they are following along exactly, their command line will be in `src` at this point and not able to run this.

Reply:
eh, you're right, but I think our users will be knowledgable enough to know where to run npm run serve from